matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/session/sync/handler/CryptoSyncHandler.kt

```diff
@@ -32,6 +33,8 @@ import org.matrix.android.sdk.internal.session.initsync.ProgressReporter
 import timber.log.Timber
 import javax.inject.Inject
 
+private val loggerTag = LoggerTag("MXMegolmEncryption", LoggerTag.CRYPTO)
```

Comment:
`MXMegolmEncryption` ? It's maybe wanted.

Reply:
good catch (bad copy paste)